src/extension/background-script/state.ts

```diff
@@ -132,10 +132,18 @@ const state = createState<State>((set, get) => ({
     } else {
       set({ mv2Password: null });
     }
-    const allTabs = browser.extension.getViews({ type: "tab" });
-    for (const tab of allTabs) {
-      tab.close();
-    }
+
+    const allTabs = await browser.tabs.query({ title: "Alby" });
```

Comment:
is this correct, we query all open tabs and check for the title?

why is the filter() on the tab ids needed?

Reply:
Follow the stackoverflow link. I think it's for TS to be happy 🤷